src/client/sandbox/node/index.js

```diff
@@ -101,4 +107,36 @@ export default class NodeSandbox extends SandboxBase {
         this.win.attach(window);
         this.element.attach(window);
     }
+
+    static processSelector (selector) {
+        var attributeSelectorRegEx = /\[([\w-^]+)=(.+?)]/g;
+        var stringifySelector      = function (value) {
+            if (value === null)
+                return 'null';
+
+            else if (value === void 0)
+                return 'undefined';
+
+            return selector;
+        };
+
+        selector = stringifySelector(selector);
+
+        return selector.replace(attributeSelectorRegEx, (str, name, value) => {
```

Comment:
`string.replace` is slower than `regex.test`  - http://jsperf.com/string-replace-versus-regex-test
(Chrome - 78%, FF - 20%, IE11 - 50%.)